src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$start-day: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$cellSize: 100px;
+$cellCollor: #eee;
+$hoverCollor: #ffbfcb;
+$cellFontHeight: 30px;
+$cellFontFamily: Arial, sans-serif;
+$gap: 1px;
+$weakSize: 7;
+$cellPading: 10px;
```

Comment:
There is a typo in the variable name `$cellPading`. It should be `$cellPadding` to follow proper spelling and the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).